### PR TITLE
COP, KES, USDT Oracle config

### PR DESCRIPTION
**Pull request by rcroessmann**

### CODE REVIEW

**Comment by sissnad** [comment]
@nvtaveras @bayological I have added `KESUSD` and `CELOUSD` configs. I have also included potential additional sources that would require either additional adapters or subscriptions

**Comment by sissnad** [comment]
I have tested them locally

**Review by nvtaveras — APPROVED** [review]

**Comment by arthurgousset** [comment]
(for visibility) 

CI on this branch is green except for a failing "[Protocol Foundry tests](https://github.com/celo-org/celo-monorepo/actions/runs/9473570208/job/26131258500?pr=10984)". 

Some context:
- failing "[Protocol Foundry tests](https://github.com/celo-org/celo-monorepo/actions/runs/9473570208/job/26131258500?pr=10984)" are not desired (obviously), but are somewhat expected.
- "[Protocol Foundry tests](https://github.com/celo-org/celo-monorepo/actions/runs/9473570208/job/26131258500?pr=10984)" are the new Foundry-based test, which we're actively developing. They are not as reliable as the existing ganache test suite, and not used in "production". In other words, we don't take passing/failing Foundry-tests as a signal of confidence.
- We still rely on ganache and other e2e tests for that, which are all passing on that branch.

TLDR: It's not a huge problem that it's failing, but I'd like to look into why the test is failing before merging the PR. 

Is this an urgent PR on your side? Is there a timeline? That way I can put some time aside to see what's wrong with the Foundry tests.

**Review by arthurgousset — CHANGES_REQUESTED** [review]
Separately, we're adopting a new release process, where we merge changes that are not on `Mainnet` into a release branch (e.g. currently [`release/core-contracts/12`](https://github.com/celo-org/celo-monorepo/tree/release/core-contracts/12)). 

We're currently doing this here: 
- https://github.com/celo-org/celo-monorepo/pull/11033

If and when a given release is audited and deployed on `Mainnet`, we rebase the release branch (`release/core-contracts/12`) onto `master` without changes to the history. 

In short, this PR should merge commits into [`release/core-contracts/12`](https://github.com/celo-org/celo-monorepo/tree/release/core-contracts/12) not `master`.

**Comment by soloseng** [comment]
Looks like the foundry tests are not passing, because of the version of celo-foundry you are using. The [celo-foundry-v0.5.13](https://github.com/celo-org/celo-foundry/tree/celo-foundry-v0.5.13) was modified so that the precompileHandler mocks expects bytes instead of bytes32 as params.

**Comment by nvtaveras** [comment]
@arthurgousset Thanks for looking into this. There's not a hard deadline but it would be nice to merge this soon as it has been opened for a while. Currently we are doing all of our oracle deployments from this branch and updating it whenever something changes.

Do you mind re-pointing the PR to the right branch and looking after it until it's merged? As on our end it's ready to go

**Comment by soloseng** [comment]
@arthurgousset I updated the base branch and fixed the failing test. This PR is set to auto-merge once you approve.

**Comment by arthurgousset** [comment]
> @arthurgousset I updated the base branch and fixed the failing test. This PR is set to auto-merge once you approve.

Thanks @soloseng! I'll wait for CI to pass, and will approve 👍 I had to re-run some tests that stalled.

**Review by arthurgousset — APPROVED** [review]
CI is passing as of https://github.com/celo-org/celo-monorepo/pull/10984/commits/7d87b9081b94ef0e9daa19c5a9fd8dac3e8f0d90. Approving on that basis ✅
